support/rally.js

```diff
@@ -55,10 +55,22 @@ module.exports = class Rally {
    */
   async _checkRallyCore() {
     try {
-      return await browser.management.get(CORE_ADDON_ID);
+      const msg = {
+        type: "core-check",
+        data: {}
+      }
+      let response =
+        await browser.runtime.sendMessage(CORE_ADDON_ID, msg, {});
+
+      if (!response
+          || response.type !== "core-check-response"
+          || response.data.enrolled !== true) {
+        return Promise.reject(
```

Comment:
Nit, but is there a difference between `return Promise.reject()` and `throw` since this is an `async` function?